mdoc/Mono.Documentation/Util/AttachedEntitiesHelper.cs

```diff
@@ -163,7 +165,8 @@ private static bool IsAttachedProperty(PropertyDefinition property, Dictionary<s
 
             return !hasExistingProperty.IsTrue() && !hasExistingField.IsTrue() &&
                 // Class X has a static field of type DependencyProperty [Name]Property
-                (property.PropertyType.Name == Consts.DependencyPropertyFullName || property.PropertyType.FullName == Consts.DependencyPropertyFullNameXaml)
+                (property.PropertyType.Name == Consts.DependencyPropertyFullName || property.PropertyType.FullName == Consts.DependencyPropertyFullNameWindowsXaml

```

Comment:
Just curious, why do we use `Name` here but `FullName` everywhere else?

Reply:
Indeed this should also be `FullName`.

Reply:
LGTM, waiting for Joel for the final review and approval.